src/rules.rs

```diff
@@ -105,9 +105,11 @@ pub(crate) fn spacing() -> SpacingDsl {
         .test("let   foo = bar;in  92", "let foo = bar; in 92")
         .inside(NODE_LET_IN).after(T![let]).single_space_or_optional_newline()
         .inside(NODE_LET_IN).around(T![in]).single_space_or_optional_newline()
-        .inside(NODE_LET_IN).before(T![in]).when(header_has_multi_value).when(before_token_not_newline).newline()
-        .inside(NODE_LET_IN).after(T![in]).when(in_body_newline).newline()
-        .inside(NODE_LET_IN).before(NODE_KEY_VALUE).when(header_has_multi_value).when(before_token_not_newline).newline()
+        .inside(NODE_LET_IN).after(NODE_KEY_VALUE).single_space_or_optional_newline()
+        .inside(NODE_LET_IN).before(NODE_KEY_VALUE).when(let_header_has_newline).newline()
+        // .inside(NODE_LET_IN).before(T![in]).when(header_has_multi_value).when(before_token_not_newline).newline()
+        .inside(NODE_LET_IN).around(T![in]).when(let_header_has_newline).newline()
+        // .inside(NODE_LET_IN).before(NODE_KEY_VALUE).when(header_has_multi_value).when(before_token_not_newline).newline()
```

Comment:
can you please remove the commented code?

Reply:
I'll clean it up.